index.html

```diff
@@ -1,258 +1,320 @@
-            <img src="assets/menu.png" alt="navbar">
-            <ul class="desktop-nav">
-                <li> <a href="">Portfolio</a></li>
-                <li> <a href="">About</a></li>
-                <li> <a href="">Contact</a></li>
-            </ul>
-        </nav>
+    <title>Cindy Kandie - Cindy's Web Development Portfolio</title>
+  </head>
+  <body>
+    <header id="item-0">
+      <div class="logo">
+        <p>Cindy K</p>
+      </div>
+      <nav>
+        <img src="assets/menu.png" alt="menu bars" />
+        <ul class="desktop-nav">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
+      </nav>
     </header>
 
-    <section class="intro-section" id= "item-1">
-        <article class="article-style">
-            <h1>I’m Cindy <br>
-                Glad to see you!
-            </h1>
-            <p>I’m a Frontend developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-        </article>
-        <div class="contact-me">
-            <h2>Let's connect</h2>
-            <ul class="socials">
-                <li>
-                    <a href="https://twitter.com/cindykandie">
-                        <img src="assets/twitter.svg" alt="My twitter page" >
-                    </a>
-                </li>
-               
-                <li>
-                    <a href="https://linkedin.com/cindykandie">
-                        <img src="assets/linkedin.png" alt="My linkedin page" >
-                    </a>
-                </li>
-                
-                <li>
-                    <a href="https://medium.com/cindykandie">
-                        <img src="assets/medium.svg" alt="My Medium page" >
-                    </a>
-                </li>
-                
-                <li>
-                    <a href="https://github.com/cindykandie">
-                        <img src="assets/github.svg" alt="My GitHub page" >
-                    </a>
-                </li>
-                
-                <li>
-                    <a href="#">
-                        <img src="assets/chill.svg" alt="My Angelist page" >
-                    </a>
-                </li>
-            </ul>
-        </div>
+    <section class="intro-section" id="item-1">
+      <article class="article-style">
+        <h1>
+          I’m Cindy <br />
+          Glad to see you!
+        </h1>
+        <p>
+          I’m a Frontend developer! I can help you build a product,
+          feature or website Look through some of my work and
+          experience! If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
+      </article>
+      <div class="contact-me">
+        <h2><a href="#contact-me"> Let's connect</a></h2>
```

Comment:
- [ ] Having a meaningful hierarchy for text headings helps your users get a sense of the page's organization and structure, it also helps guide the eye around the page 😊.